gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,4 @@
-#Sun May 07 08:40:37 PDT 2017
+#Sat Jun 17 07:21:26 PDT 2017
```

Comment:
It's autogenerated. And it documents when the file was generated.
I don't think we should manually edit this file.